### DISCUSSION TITLE
DSO markers without names?

### DISCUSSION THREAD

**Discussion by turboscrew** [Stellarium/stellarium#2328] [Q&A] [answered]
Is there a way to keep the DSO markers without the DSO names?
The names do clutter the view, and you can get the DSO info displayed on the upper left corner by selecting the DSO marker.

**Comment by gzotti** [comment, marked as answer]
There are two sliders in the View Settings/DSO tab. One is called "Labels", the other is called "Hints". I leave it to your experimentation to find out what they mean.
